lib/async.js

```diff
@@ -1,9 +1,11 @@
+/* eslint-disable max-lines */
 var fs = require('fs');
 var path = require('path');
 var caller = require('./caller.js');
 var nodeModulesPaths = require('./node-modules-paths.js');
 var normalizeOptions = require('./normalize-options.js');
 var isCore = require('is-core-module');
+var resolveExports = require('./resolve-imports-exports.js');
```

Comment:
```suggestion
var resolveExports = require('./resolve-imports-exports');
```
require paths should always omit extensions.

Reply:
Should I change the other require calls to match?

Reply:
No need; I've done that in a54c67e / a112d1cc3b65f827149cf509833428b8a90034d6